tests/python/pants_test/net/http/test_fetcher.py

```diff
@@ -378,4 +379,4 @@ def test_download_redirect(self):
       self.assertTrue(self._URL1_ACCESSED)
 
       with open(path) as fp:
-        self.assertEqual('returned from redirect\r\n', fp.read())
+        self.assertIn(fp.read(), ['returned from redirect\n', 'returned from redirect\r\n'])
```

Comment:
In Py2, HTTP Server adds `\r\n` and in Py3 just `\n`. I'm concerned this may be a platform specific implementation, which is why I changed this test to assertIn, rather than using an `if PY3` check